sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -176,7 +176,7 @@ public void connectToService(Context context, Core core, MessagePipeHandle pipe)
         registry.register(HapticFeedback.MANAGER.getName(), new ServiceFactory() {
             @Override
             public void connectToService(Context context, Core core, MessagePipeHandle pipe) {
-                HapticFeedback.MANAGER.bind(new HapticFeedbackImpl(), pipe);
+                HapticFeedback.MANAGER.bind(new HapticFeedbackImpl((android.app.Activity) context), pipe);
```

Comment:
What is the lifetime of the service registry?  Is it global to the application, or is there one registry per activity?